test/src/module/command/expireTest.c

```diff
@@ -134,7 +129,7 @@ testRun(void)
         "\"backup-archive-stop\":\"000000010000000000000011\",\"backup-info-repo-size\":2369186,"
         "\"backup-info-repo-size-delta\":346,\"backup-info-size\":20162900,\"backup-info-size-delta\":8428,"
         "\"backup-prior\":\"20181119-152900F\",\"backup-reference\":[\"20181119-152900F\"],"
-        "\"backup-timestamp-start\":%" PRIu64 ",\"backup-timestamp-stop\":%" PRIu64 ",\"backup-type\":\"diff\","
+        "\"backup-timestamp-start\":%" PRIu64 ",\"backup-timestamp-stop\":%" PRIu64 ",\"backup-type\":\"incr\","
```

Comment:
This should be diff.

Reply:
Yep. Made the change in the wrong spot. It isn't used in the tests - just nice to have the label match the type. Fixed in two places in https://github.com/pgbackrest/pgbackrest/pull/1420/commits/334c502225450dc7215f27369d89bd1feb840eeb